### ISSUE TITLE
Error in account_management/create_customer.py Example

### ISSUE THREAD

**Issue by BenRKarl** [googleads/google-ads-python#158]
When running this example using `'v2'` the below error is generated:

```
TypeError: create_customer_client() missing 2 required positional arguments: 'email_address' and 'access_role'
```

Seems the signature for the `CustomerService.CreateCustomerClient` method changed and needs to be updated in the example.

**Comment by asafzinger** [comment]
I tried to alter the code like this:
 ```

role = client.get_type('AccessRoleEnum').ADMIN
response = customer_service.create_customer_client(mgr_id, customer, "mail@gmail.com", role)

```
But now I'm getting this error:
```
 File <>/.local/share/virtualenvs/ml-googleapi-n6J6a0FH/lib/python3.7/site-packages/google/ads/google_ads/v2/services/customer_service_client.py", line 390, in create_customer_client
    access_role=access_role,
TypeError: Parameter to MergeFrom() must be instance of same class: expected google.protobuf.StringValue got str.
```
Can you please share a working solution?

**Comment by BenRKarl** [comment]
@asafzinger We're working on updating the example. Meanwhile that error is probably coming from the third parameter, which should be a [StringValue](https://developers.google.com/protocol-buffers/docs/reference/csharp/class/google/protobuf/well-known-types/string-value.html#class_google_1_1_protobuf_1_1_well_known_types_1_1_string_value) not a raw string, so you can do something like this:

```python
email = client.get_type('StringValue', version='v2')
email.value = 'mail@gmail.com'
...
response = customer_service.create_customer_client(mgr_id, customer, email, role)
```

**Comment by BenRKarl** [comment]
@asafzinger Turns out I was misunderstood in my initial assessment. The `create_customer_client` method was not updated, these two parameters were just marked as required when they are not. I fixed the problem in this PR: https://github.com/googleads/google-ads-python/pull/162

**Comment by BenRKarl** [comment]
@asafzinger This has been fixed in v3.1.2 of this library, would you please update and try again?

**Comment by asafzinger** [comment]
@BenRKarl just verified that it is solved in v3.1.2.
Thank you so much
